frontend/src/components/Connect/Details/Actions/Actions.tsx

```diff
@@ -87,99 +89,155 @@ const Actions: React.FC = () => {
         }
       >
         {connector?.status.state === ConnectorState.RUNNING && (
-          <ActionDropdownItem
+          <ActionDropdownItemWithFallback
             onClick={pauseConnectorHandler}
-            permission={{
-              resource: ResourceType.CONNECT,
-              action: Action.OPERATE,
-              value: routerProps.connectName,
-            }}
+            permission={[
+              {
+                resource: ResourceType.CONNECTOR,
+                action: Action.OPERATE,
+                value: connectorPath,
+              },
+              {
+                resource: ResourceType.CONNECT,
+                action: Action.OPERATE,
+                value: routerProps.connectName,
+              },
+            ]}
           >
             Pause
-          </ActionDropdownItem>
+          </ActionDropdownItemWithFallback>
         )}
         {connector?.status.state === ConnectorState.RUNNING && (
-          <ActionDropdownItem
+          <ActionDropdownItemWithFallback
             onClick={stopConnectorHandler}
-            permission={{
-              resource: ResourceType.CONNECT,
-              action: Action.OPERATE,
-              value: routerProps.connectName,
-            }}
+            permission={[
+              {
+                resource: ResourceType.CONNECTOR,
+                action: Action.OPERATE,
+                value: connectorPath,
+              },
+              {
+                resource: ResourceType.CONNECT,
```

Comment:
Let's introduce fallbackPermission param within ActionDropdown and check it there instead of array